pypdf/_page.py

```diff
@@ -765,7 +765,7 @@ def _get_inline_images(self) -> Dict[str, ImageFile]:
                 )
             elif ope in (b"BI", b"EI", b"ID"):  # pragma: no cover
                 raise PdfReadError(
-                    f"{ope} operator met whereas not expected,"
+                    f"{ope!r} operator met whereas not expected,"
```

Comment:
We need this because otherwise I get
```python
$ mypy pypdf/
pypdf/_page.py:768: error: If x = b'abc' then f"{x}" or "{}".format(x) produces "b'abc'", not "abc". If this is desired behavior, use f"{x!r}" or "{!r}".format(x). Otherwise, decode the bytes  [str-bytes-safe]
Found 1 error in 1 file (checked 52 source files)
```